test/Dapr.Client.Test/PublishEventApiTest.cs

```diff
@@ -87,7 +89,7 @@ public async Task PublishEventAsync_CanPublishTopicWithNoContent()
 
             request.PubsubName.Should().Be(TestPubsubName);
             request.Topic.Should().Be("test");
-            jsonFromRequest.Should().Be("\"\"");
```

Comment:
Do we need a UT for WithContent_ContentEqualsNull?

Reply:
If you mean a for passing in `null` as a value, I don't mind having one. The thing is .. this isn't testing what our code does, its mostly testing what our code DOESN'T do 😆 I don't mind having one test, but I don't want to go on an extensive journey of testing the underlying behaviors of the JSON serializer.

Reply:
Yes, passing null as the value is the ut I am suggesting since that's the case we found was broken and also establishes the behavior of the API if someone is looking. I agree that writing tests that are more like json serializer tests is unnecessary.